Pear/Profile/EditProfileViewController.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  EditProfileViewController.swift
+//  Pear
+//
+//  Created by Mathew Scullin on 4/17/22.
+//  Copyright © 2022 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class EditProfileViewController: UIViewController {
+    
+    // MARK: - Private View Vars
+    private let backButton = UIButton()
+    private let dotsView = DotsView()
+    private var editOptions = [UIView]()
+    private var profileSectionsCollectionView: FadeWrapperView<UICollectionView>!
+    private var saveButton = UIButton()
+    
+    // MARK: - Private Data Vars
+    private var currentUser: UserV2
+    private var profileSections = [ProfileSectionType]()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        view.backgroundColor = .backgroundLightGreen
+
+        navigationController?.navigationBar.isHidden = false
+        navigationController?.navigationBar.barTintColor = .backgroundLightGreen
+        navigationController?.navigationBar.shadowImage = UIImage() // Hide navigation bar bottom shadow
+        navigationController?.navigationBar.titleTextAttributes = [
+            .font: UIFont._24CircularStdMedium,
+            .foregroundColor: UIColor.primaryText
+        ]
+        navigationItem.title = "Preview"
+        
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)
+        
+        saveButton.setTitle("Save", for: .normal)
+        saveButton.setTitleColor(.darkGreen, for: .normal)
+        saveButton.titleLabel?.font = ._20CircularStdMedium
+        saveButton.addTarget(self, action: #selector(savePressed), for: .touchUpInside)
+        navigationItem.rightBarButtonItem = UIBarButtonItem(customView: saveButton)
+        
+        view.addSubview(dotsView)
+        
+        let profileEditCollectionViewLayout = UICollectionViewFlowLayout()
```

Comment:
why use a horizontal scrolling collectionview instead of a uipageviewcontroller?